### DISCUSSION TITLE
Is up node the same with parent node? Is same-level node the same with sibling node?

### DISCUSSION THREAD

**Discussion by ooker777** [SkepticMystic/breadcrumbs#308] [Q&A] [answered]
Similarly, is a sibling node a same-level node? This seems obviously, but I want to make sure that the [comprehensions](https://en.wikipedia.org/wiki/Comprehension_(logic) "Comprehension (logic) - Wikipedia") of the two concepts are the same. 

Let's A a next node of B. It is implied that B is a previous node of A. Are A and B sibling nodes?

**Comment by michaelpporter** [comment, marked as answer]
Short answer: yes, by convention — but the names are just labels, not fixed meanings.

The five default fields each represent a **direction**, and most people read them like this:

| Field  | Conventional reading         |
| ------ | ---------------------------- |
| `up`   | parent / broader note        |
| `down` | child / narrower note        |
| `same` | sibling                      |
| `next` | following note in a sequence |
| `prev` | preceding note in a sequence |

Breadcrumbs itself attaches no inherent "parent"/"sibling" semantics — you can rename these fields, delete them, or add your own (`parent`, `child`, `day`, `month`, …) and group them however you like. The only built-in behaviour is the default implied reverse rules: `up`↔`down`, `next`↔`prev`, and `same`↔`same`.

This is now written up on the docs: https://breadcrumbs-docs.michaelpporter.com/edge-fields/ 🍞
